src/answers-umd.js

```diff
@@ -186,7 +186,23 @@ class Answers {
       },
       persistedValueParser: this._parsePersistentStorageValue
     });
-    storage.init(window.location.search);
+
+    /**
+     * Guard the case of user copying a link with query-id param to a new tab
+     * and firing a FOLLOW_UP_QUERY analytics event from their first search.
+     * This is different from redirecting a href click into a new tab or a page
+     * refresh, which should fire a FOLLOW_UP_QUERY analytics event upon next
+     * search on load.
+     */
+    const searchParams = new SearchParams(window.location.search);
+    if (searchParams.has(StorageKeys.QUERY_ID)) {
+      const isPageReload = window.performance.getEntriesByType('navigation')?.[0]?.type === 'reload';
+      if (!document.referrer && !isPageReload) {
+        searchParams.delete(StorageKeys.QUERY_ID);
```

Comment:
Is there a chance for the url to get out of sync with persistent storage?

Reply:
Are you saying if the persistent storage would store a set of a different params from the modified url here? The storage is initialized based on the searchParams [here after it is modified ](https://github.com/yext/answers-search-ui/pull/1727/files/3a73dd4b18891862fa697ece1c5383c507d920ab#diff-151f0f77fa3617c80e488938e18494f85ca7eb1cea04bc981728ce3408ff10f2R205) so I don't think that would be an issue